dev/docker-compose.yaml

```diff
@@ -31,8 +37,10 @@ services:
       - full
     restart: unless-stopped
     volumes:
-      - ../apisix/apisix.yaml:/usr/local/apisix/conf/apisix.yaml:ro
-      - ./dapr.yaml:/usr/local/apisix/dapr.yaml:ro
+      - ./dapr.yaml:/usr/local/apisix/dapr/config.yaml:ro
+      - ../apisix/apisix.yaml:/usr/local/apisix/conf/apisix.yaml:ro # for hot-reloading
+    environment:
+      ORY_WEBHOOK_BASIC_AUTH: '{"username":"ory","password":"2341"}'
```

Comment:
we should probably document all values in this file in a README for this directory

Reply:
I've added a `.env.example` into the `apisx/` directory. The other `SECRETSTORE_` envs are documented inside the `README.md`.
Yes, we should document our current Dapr and APISIX workflows/setup when the config/setup is no longer so volatile.